webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/plugins/UpgradePluginPR3383.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.store.triplestore.upgrade.plugins
+
+import org.eclipse.rdf4j.sparqlbuilder.core.SparqlBuilder.`var` as variable
+import org.eclipse.rdf4j.sparqlbuilder.core.query.*
+import org.eclipse.rdf4j.sparqlbuilder.core.query.ModifyQuery
+import org.eclipse.rdf4j.sparqlbuilder.graphpattern.TriplePattern
+
+import org.knora.webapi.slice.admin.AdminConstants
+import org.knora.webapi.slice.common.repo.rdf.Vocabulary
+import org.knora.webapi.slice.common.repo.rdf.Vocabulary.KnoraAdmin as KA
+import org.knora.webapi.store.triplestore.upgrade.GraphsForMigration
+import org.knora.webapi.store.triplestore.upgrade.MigrateSpecificGraphs
+
+/**
+ * Configuring default object access permissions on the SystemProject is not supported anymore.
+ * This plugin removes all default object access permissions for the SystemProject.
+ */
+class UpgradePluginPR3383 extends AbstractSparqlUpdatePlugin {
+
+  override def graphsForMigration: GraphsForMigration =
+    MigrateSpecificGraphs.from(AdminConstants.permissionsDataNamedGraph)
+
+  private val removeSystemProjectDefaultObjectAccessPermissions: ModifyQuery = {
```

Comment:
I just noticed, that this upgrade should also remove existing configurations on the `KnownUser` group.

Apart from this missing query the PR is RfR.

Reply:
=> 0977eeb87aef771aed27aa9335e3a7a88d817ba0